src/lib/layer-manager.js

```diff
@@ -142,19 +141,20 @@ export default class LayerManager {
     let redraw = false;
 
     // Make sure that buffer is cleared once when layer list becomes empty
-    if (this.layers.length === 0 && this.drewLayers) {
-      redraw = true;
-      return true;
-    }
-
-    if (this.redrawNeeded) {
-      this.redrawNeeded = false;
-      redraw = true;
+    if (this.layers.length === 0) {
+      if (this.layerCleared === false) {
+        redraw = true;
```

Comment:
I realize that this is a relic from before the change, but looks like we are setting `redraw` but not using it before we return.